src/analytics/navigationTracker/navigationTracker.ts

```diff
@@ -154,6 +156,12 @@ export class NavigationTracker extends BaseTracker {
    */
   isHeaderMenuBarLink(element: HTMLElement): boolean {
     const header = document.getElementsByTagName("header")[0];
-    return header.contains(element);
+    return header && header.contains(element);
+  }
+
+  isPhaseBannerLink(element: HTMLElement): boolean {
```

Comment:
Well Done!
This function will be very useful later, when we will have to deal with the section property.